server/routes/maintainingReports/coordinator.ts

```diff
@@ -13,9 +13,59 @@ export default class CoordinatorRoutes {
     private readonly involvedStaffService: InvolvedStaffService,
     private readonly reviewService: ReviewService,
     private readonly offenderService: OffenderService,
-    private readonly systemToken: SystemToken
+    private readonly systemToken: SystemToken,
+    private readonly userService: UserService
   ) {}
 
+  viewRemovalRequest: RequestHandler = async (req, res) => {
+    const { reportId, statementId } = req.params
+    const token = await this.systemToken(res.locals.user.username)
+    const [{ name, userId, email }, removalReason] = await Promise.all([
+      this.involvedStaffService.loadInvolvedStaff(parseInt(reportId, 10), parseInt(statementId, 10)),
+      this.involvedStaffService.getInvolvedStaffRemovalRequestedReason(parseInt(statementId, 10)),
+    ])
+    const location = await this.userService.getUserLocation(token, userId)
+    const data = {
+      name,
+      userId,
+      location,
+      email,
+      removalReason,
+    }
+    const errors = req.flash('errors')
+    res.render('pages/coordinator/view-removal-request.html', { data, errors })
+  }
+
+  submitRemovalRequest: RequestHandler = async (req, res) => {
+    const { reportId, statementId } = req.params
+    const { confirm } = req.body
+    const endpoint = `/coordinator/report/${reportId}/statement/${statementId}`
```

Comment:
would be nice to move this to `config/incident.paths`...

Reply:
Done